casr/src/bin/casr-cli.rs

```diff
@@ -906,23 +906,23 @@ fn print_summary(dir: &Path, unique_crash_line: bool) {
             continue;
         }
 
-        println!("==> <{}>", filename.magenta());
+        eprintln!("==> <{}>", filename.magenta());
```

Comment:
Is it OK that we changed to `eprintln!`? What's the reason? Does it preserve backward compatibility?

Reply:
changed to println